pom.xml

```diff
@@ -323,7 +323,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v1/_openapi.json.tpl</inputSpec>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/e87e1ff9bdedb8339a160a0741f0b2d70a4dbdd4/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v1/_openapi.json.tpl</inputSpec>
```

Comment:
remember to restore this ref

Reply:
Fixed with 992f63cbfd184f9c0b084e3a481694a076fc02d5